modules/govuk/manifests/node/s_base.pp

```diff
@@ -34,8 +34,19 @@
     include hosts
   }
 
-  $app_classes = regsubst($apps, '^', 'govuk::apps::')
-  include $app_classes
+  if $::aws_migration {
+    $node_class = $::aws_migration
+  } else {
+    $node_class = regsubst($::fqdn_short, '-.*$', "")
```

Comment:
Should we use the `govuk_node_class` ?

Reply:
I chose not to use this because this is used within the main manifest and thought it might be misleading.